package.json

```diff
@@ -30,4 +30,4 @@
     "typescript-eslint": "^8.18.2",
     "vite": "^6.0.5"
   }
-}
+}
```

Comment:
always add newline at the end of file.

If you are using vscode: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files